context/app/static/js/shared-styles/tables/SelectableRowCell/SelectableRowCell.jsx

```diff
@@ -1,23 +1,31 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 
 import Checkbox from '@material-ui/core/Checkbox';
 import TableCell from '@material-ui/core/TableCell';
 
 import { useStore } from 'js/shared-styles/tables/SelectableTableProvider/store';
 
-function SelectableRowCell({ rowKey, index, disabled }) {
+function SelectableRowCell({ rowKey, disabled }) {
   const { toggleRow, selectedRows, tableLabel } = useStore();
 
   return (
     <TableCell padding="checkbox">
       <Checkbox
         checked={selectedRows.has(rowKey)}
-        inputProps={{ 'aria-labelledby': `${tableLabel}-row-${index}-checkbox` }}
+        inputProps={{ 'aria-label': `${tableLabel}-row-${rowKey}-checkbox` }}
         onChange={() => toggleRow(rowKey)}
         disabled={disabled}
       />
     </TableCell>
   );
 }
 
+SelectableRowCell.propTypes = {
+  /**
+   Unique key representing the table.
+  */
+  rowKey: PropTypes.string.isRequired,
+};
```

Comment:
I'm ok with not having proptypes... but if we do have them, should optional `disabled` be included?

Reply:
Added, thanks!